### PR TITLE
PLAT-11223 updated default java and python bdk versions

### PR DESCRIPTION
**Pull request by symphony-soufiane**
### Ticket
[PLAT-11223](https://perzoinc.atlassian.net/browse/PLAT-11223)

### Description
Updated default java and python bdk versions after releasing Java BDK 2.2.1 and Python Legacy 1.3.6

### CODE REVIEW

**Review by symphony-elias — APPROVED** [review]

**Comment by symphony-youri** [comment]
Don't we need to change SPRING_VERSION_DEFAULT?

**Review by symphony-youri — CHANGES_REQUESTED** [review]
SPRING_VERSION_DEFAULT should probably be aligned with the one from the BDK

**Review by symphony-youri — APPROVED** [review]

**Comment by symphony-youri** [comment]
It fails because of    > Spring Boot plugin requires Gradle 6.8.x, 6.9.x, or 7.x. The current version is Gradle 6.6.1
 we just need to update the gradle wrapper

File: generators/bdk/java/templates/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.6.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.0-bin.zip
```
Comment (symphony-soufiane) [inline generators/bdk/java/templates/gradle/wrapper/gradle-wrapper.properties:3]: This upgrade was required when upgraded spring default version to 2.5.3

**Review by symphony-elias — APPROVED** [review]
LGTM

File: generators/python-bots/templates/python/bots/elements_form/requirements.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-sym-api-client-python>=0.1.18
+sym-api-client-python>=1.3.6
```
Comment (symphony-youri) [inline generators/python-bots/templates/python/bots/elements_form/requirements.txt:1]: ohhh that was an old version, have you tried to generate it and checked the bot is running?

**Review by symphony-youri — APPROVED** [review]

**Pull request merged**